console/v1/types_console_link.go

```diff
@@ -0,0 +1,37 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Extension for customizing OpenShift web console links
+type ConsoleLink struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+	Link              `protobuf:"bytes,2,opt,name=link"`
+	// menu determines which location in the console the link will be appended to
+	Location ConsoleLocationSelector `json:"menu" protobuf:"bytes,3,opt,name=menu,casttype=ConsoleMenuSelector"`
+}
+
+// ConsoleMenuSelector is a set of possible menu targets to which a
+// link may be appended.  Links will be ordered alphabetically.
+type ConsoleLocationSelector string
+
+const (
+	// the help menu in the console
```

Comment:
My original comment disappeared 😕

Reply:
`helpMenu` & `userMenu` work.  Or we could `help menu`, `help.menu`, etc.  Any care here, from a user api perspective?
@spadgett @rhamilto